src/components/TagInput/TagInput.props.ts

```diff
@@ -0,0 +1,16 @@
+import { InputHTMLAttributes } from 'react';
+
+import type { TagInputConfig } from './TagInput.styles';
+
+import { BasicInputState } from '@/types';
+
+export type TagInputProps = {
+  custom?: TagInputConfig;
+  state?: BasicInputState;
+  isValidationError?: boolean;
+  label?: string;
```

Comment:
It's should be `Pick<LabelProps, 'label' | 'action' | 'optional'>`

<img width="181" alt="image" src="https://github.com/VirtusLab/tetrisly-react/assets/6291752/e7f351da-d76b-40b0-b6f2-3eed9b6fa8c0">

Reply:
And what about Tooltip?

Reply:
done